src/bun.js/net.exports.js

```diff
@@ -368,7 +380,206 @@ export function createConnection(port, host, connectListener) {
 
 export const connect = createConnection;
 
+class Server extends EventEmitter {
+  #connectionListener;
+  #options;
+  #server;
+  #listening;
+  #connections;
```

Comment:
Can you add `maxConnections = 0` here? That tells the engine what fields to expect

Reply:
done on https://github.com/oven-sh/bun/pull/2337/commits/dc6bc3c6d8d73d3f5588116eeeaab409d01c765f